base_phone_validation/README.rst

```diff
@@ -0,0 +1,8 @@
+Base Phone Validation
+=====================
+Force phone numbers to be valid against the the country of the partner.
+
+Contributors
+------------
+* Jordi Riera <kender.jr@gmail.com>
+
```

Comment:
For the file README.rst, you should use the official OCA template available here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst